maquetado/scss/_tech.scss

```diff
@@ -52,15 +40,10 @@
 }
 
 .tech-list {
-  display: flex;
-  flex-wrap: wrap;
-  padding-left: 100px;
-
-  &:not(:last-child) {
-    margin-bottom: 100px;
-  }
-
-  .tech {
-    margin: 20px;
-  }
+  align-items: center;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(3, $tech-width);
```

Comment:
```suggestion
  grid-template-columns: repeat(3, 1fr);
```
For the responsive 😉

Reply:
Si agrego tu sugerencia, pasa esto:

<img width="1679" alt="Screen Shot 2019-07-22 at 9 36 25 AM" src="https://user-images.githubusercontent.com/22043902/61641161-42dd0980-ac64-11e9-9c17-257a868e340e.png">

Reply:
Yes, my mistake. I'm sorry